tools/interactive_marker_teleop/include/interactive_marker_teleop.h

```diff
@@ -0,0 +1,65 @@
+/* Copyright (c) 2017, United States Government, as represented by the
+ * Administrator of the National Aeronautics and Space Administration.
+ *
+ * All rights reserved.
+ *
+ * The Astrobee platform is licensed under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with the
+ * License. You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+#ifndef INTERACTIVE_MARKER_TELEOP_H_
+#define INTERACTIVE_MARKER_TELEOP_H_
+
+#include <ff_msgs/CommandStamped.h>
+#include <ff_msgs/CommandConstants.h>
+#include <ff_util/ff_names.h>
+
+#include <ros/ros.h>
+#include <interactive_markers/interactive_marker_server.h>
+#include <interactive_markers/menu_handler.h>
+#include <tf2_ros/transform_listener.h>
+#include <tf2_geometry_msgs/tf2_geometry_msgs.h>
+
+#include <string>
+
+namespace vm = visualization_msgs;
```

Comment:
Not very important in this case since nothing will be including this header file other than the .cpp but the general rule of thumb is to put the "namespace a = b;" alias only in the cpp files so other files that include this header don't accidentially use this namespace alias that they might not be aware of.  The unfortunate side effect is not being able to use namespace aliases in headers, so vm:: would be replaced with visualization_msgs::  for this file (but not the cpp).